theme/templates/resource-landing-page/top-right-buttons.html

```diff
@@ -92,7 +92,9 @@
                         {% csrf_token %}
 
                         {% if page.perms.delete or cm.raccess.published %}
-                            <a id="new-version" data-toggle="modal" data-target="#new-version-resource-dialog">
+                            <a id="new-version" {% if cm.locked_time is None %}data-toggle="modal"{% endif %}
+                               data-target="#new-version-resource-dialog"
+                               {% if cm.locked_time is not None %}class="disabled"{% endif %}>
```

Comment:
And this will prevent someone from refreshing the page to re-enable the new version button while the new version is being created. Looks good.  Noting that the new version functionality is locked on the backend too.